.github/workflows/001-tester-ubuntu-make-test-auraed.yml

```diff
@@ -63,38 +63,9 @@ jobs:
         with:
           toolchain: stable
           components: clippy, rustfmt
-      - name: Install protoc-gen-doc in [ubuntu:latest]
-        run: |
-          wget https://github.com/pseudomuto/protoc-gen-doc/releases/download/v1.5.1/protoc-gen-doc_1.5.1_linux_amd64.tar.gz
-          tar -xzf protoc-gen-doc_1.5.1_linux_amd64.tar.gz
-          chmod +x protoc-gen-doc
-          cp protoc-gen-doc /usr/local/bin/protoc-gen-doc
-          sudo apt-get update
-          sudo apt-get install -y protobuf-compiler
-      - name: Install compile time dependencies (musl-tools) in [ubuntu:latest]
-        #
-        # Do not add GNU libraries here! Do not add GNU libraries here!
-        #
-        # Please (seriously please) be careful about adding commands here.
-        # This is our core way of validating that our binary is "healthy"
-        # If we need to install anything with the word "lib" in it to get
-        # the build to pass, we likely should be having other discussions
-        # instead of adding commands here.
-        #
-        # Do not add GNU libraries here! Do not add GNU libraries here!
-        #
-        # For example, we should NOT be adding libraries such as "libseccomp"
-        # or "libdbus".
-        #
-        # If in doubt, please ask in Discord in the build channel.
-        #
-        # Do not at GNU libraries here! Do not add GNU libraries here!
-        run: |
-          sudo apt-get update
-          sudo apt-get install -y musl-tools
       - name: Cargo build, lint, test [make test]
         # This should remain the only command we execute as this matches the title of the file.
         # The goal is for this to be easy to find from the GitHub dashboard.
         # Instead of adding more commands here, consider another make target or a new YAML file
         # named with a good name.
-        run: make auraed-test
+        run: make os-deps auraed-test
```

Comment:
should auraed-test just depends on os-deps?  or on auraed-build which depends on os-deps... 

that would make things easier for people getting started I think.

Reply:
hrm I'm not sure what the best ux is. Most likely `os-deps` needs to get added to one or more of the super targets, but not sure which. Thoughts?

Reply:
leave this as it is (and yes please update the other workflows) and I'll see what I can do.